automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/RuntimeFixtureSpec.scala

```diff
@@ -23,7 +24,7 @@ abstract class RuntimeFixtureSpec
     with LeonardoTestUtils
     with Retries {
 
-  implicit val ronToken: AuthToken = ronAuthToken
+  implicit def ronToken: AuthToken = Ron.authToken()
```

Comment:
Do you need this `def`? Could just call `Ron.authToken()` at each call site.

Reply:
Not needed but removing it would increase the diff in this PR quite a bit since that `implicit` is relied on indirectly in a lot of places where `RuntimeFixtureSpec` is mixed in. I was hoping to contain the diffs in this PR and have a follow-up tech debt ticket do more of the refactoring so we can get the fix in sooner and with less chance of introducing errors. What do you think?